react/tic-tac-toe/src/redux/users/actions.js

```diff
@@ -13,6 +13,7 @@ const actionsCreators = {
   login: userData => async dispatch => {
     const response = await usersService.userLogin(userData);
     if (response.ok) {
+      localStorage.setItem('token', response.data);
```

Comment:
we should be using this to initialize the state, so when the app is reinitialized, the token is taken from local storage and the user is automatically logged in if there's a token there